.env.base

```diff
@@ -46,7 +46,6 @@ REACT_APP_FEATURE_LIFI_SWAP=true
 REACT_APP_FEATURE_ONE_INCH=true
 REACT_APP_FEATURE_THOR_SWAP=true
 REACT_APP_FEATURE_THOR_SWAP_STREAMING_SWAPS=true
-REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES=false
```

Comment:
This was set to an old value of 25 and wasn't effectively used since FOX donations. Made us do some weird branching to accomodate for THOR and made things v. v. hard to grasp